test/suites/snapshot.robot

```diff
@@ -83,3 +100,10 @@ Extend LVMD Config
     ${patch_cfg}=    OperatingSystem.Get File    ${LVMD_PATCH}
     ${merged_cfg}=    Lvmd Merge    ${default_cfg}    ${patch_cfg}
     Upload String To File    ${merged_cfg}    /etc/microshift/lvmd.yaml
+
+Delete LVMD Config
```

Comment:
Should this work more like the MicroShift config and restore the "old" file contents? If not, maybe the `Lvmd Merge` keyword can be simplified, too?

Reply:
The latter is probably the best approach.  I'll rework it to just write a hardcoded lvmd to disk without worrying about what's in the configMap.  Unless we add additional storage tests, there's no need to restore the lvmd - it's not there to begin with. It's enough to delete the file and restart the service.